### PR TITLE
Jail Time

### PR DESCRIPTION
**Pull request by yeiij**
## Changes proposed in this pull request:

• Adds the option to set jail time

### CODE REVIEW

**Comment by Jawshy** [comment]
Ref: https://github.com/AsYetUntitled/Framework/pull/89

**Comment by tkcjesse** [comment]
Limits should be optional in M_Config, also a way to enable/disable time setting in general via M_Config.

File: Altis_Life.Altis/stringtable.xml [hunk 1]
```diff
@@ -11670,23 +11670,58 @@
         </Key>
         <Key ID="STR_Jail_Text">
             <Original>Jail time</Original>
+            <Czech></Czech>
```
Comment (SimZor) [inline Altis_Life.Altis/stringtable.xml]: doba vězení

File: Altis_Life.Altis/stringtable.xml [hunk 2]
```diff
@@ -11670,23 +11670,58 @@
...
+            <French></French>
```
Comment (SimZor) [inline Altis_Life.Altis/stringtable.xml]: Temps de prison

File: Altis_Life.Altis/stringtable.xml (hunk 1 above)
Comment (SimZor) [inline Altis_Life.Altis/stringtable.xml]: doba vězení

File: Altis_Life.Altis/stringtable.xml [hunk 3]
```diff
@@ -11670,23 +11670,58 @@
...
             <Spanish>Tiempo de carcel</Spanish>
+            <Italian></Italian>
```
Comment (SimZor) [inline Altis_Life.Altis/stringtable.xml]: Periodo di detenzione

File: Altis_Life.Altis/stringtable.xml [hunk 4]
```diff
@@ -11670,23 +11670,58 @@
...
+            <Polish></Polish>
```
Comment (SimZor) [inline Altis_Life.Altis/stringtable.xml]: czas Jail

File: Altis_Life.Altis/stringtable.xml [hunk 5]
```diff
@@ -11670,23 +11670,58 @@
...
+            <Portuguese></Portuguese>
```
Comment (SimZor) [inline Altis_Life.Altis/stringtable.xml]: Tempo de prisão

File: Altis_Life.Altis/stringtable.xml [hunk 6]
```diff
@@ -11670,23 +11670,58 @@
...
+            <Russian></Russian>
```
Comment (SimZor) [inline Altis_Life.Altis/stringtable.xml]: тюремное заключение

File: Altis_Life.Altis/stringtable.xml [hunk 7]
```diff
@@ -11670,23 +11670,58 @@
...
+            <German></German>
```
Comment (SimZor) [inline Altis_Life.Altis/stringtable.xml]: Zeit im Gefängnis

**Comment by BoGuu** [comment]
Need a database link with this if it has a chance of getting pulled.

**Comment by tkcjesse** [comment]
Uh? Why close?

**Comment by yeiij** [comment]
The pull request  is rejected due to no DB implementation, maybe i will do it in a future.

**Pull request closed without merging**